### ISSUE TITLE
ModelSerializer test failure against Django `master`

### ISSUE THREAD

**Issue by carltongibson** [tomchristie/django-rest-framework#3428]
[See build failure here](https://travis-ci.org/tomchristie/django-rest-framework/jobs/81584238#L172)

```
_________________ TestRegularFieldMappings.test_regular_fields _________________
tests/test_model_serializer.py:168: in test_regular_fields
    self.assertEqual(unicode_repr(TestSerializer()), expected)
E   AssertionError: "Test[446 chars]its=3, validators=[<django.core.validators.Dec[616 chars]ld>)" != "Test[446 chars]its=3)\n    email_field = EmailField(max_lengt[553 chars]ld>)"
E   Diff is 1206 characters long. Set self.maxDiff to None to see it.
```

@tomchristie:

> The failure there should be easy enough to resolve - something on ModelSerializer is being mapped slightly diff between versions. Would be good to fully see the diff between the actual repr() and the expected repr() — https://github.com/tomchristie/django-rest-framework/pull/3427#issuecomment-142288865

**Comment by tomchristie** [comment]
We can update this to 1.9-alpha now :smile:

**Comment by carltongibson** [comment]
Is that today? Wow.

**Comment by tomchristie** [comment]
Resolved via #3450.
